tests/libwait/wait.go

```diff
@@ -188,6 +189,11 @@ func WaitUntilVMIReady(vmi *v1.VirtualMachineInstance, loginTo console.LoginToFu
 		opts...,
 	)
 	gomega.Expect(loginTo(vmi)).To(gomega.Succeed())
+
+	var err error
```

Comment:
Why is it necessary to refresh the VMI object?
How does the VMI object changes after the login is done?